assets/dev/js/editor/elements/models/widget.js

```diff
@@ -0,0 +1,8 @@
+import ElementModel from './element';
+
+export default class Widget extends ElementModel {
+	isValidChild() {
+		// By default widget cannot contain any element.
+		return false;
```

Comment:
Maybe put that in the Base, so every element can't get any child unless explicitly set?